util/check-format-commit.sh

```diff
@@ -164,7 +164,7 @@ done
 cat $TEMPDIR/results-filtered.txt
 
 # If any findings were in range, exit with a different error code
-if [ -n $TEMPDIR/results-filtered.txt ]
+if [ -n "$(cat $TEMPDIR/results-filtered.txt)" ]
```

Comment:
Do you want `-s $TEMPDIR/results-filtered.txt` here? (File exists with size > 0)

Reply:
Ah, yeah that's probably better

Reply:
Done